src/shared/codelens/localLambdaRunner.ts

```diff
@@ -667,32 +678,31 @@ async function waitForDebugPort({
     debugPort,
     configuration,
     channelLogger,
+    timeoutDuration
```

Comment:
For parity with the previous code, timeoutDuration should have a default of `SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT`

Reply:
We're initially setting that as the default for the entire trial run (see line 521 of the current revision). Is there a reason we'd potentially want to reset this value?

Reply:
That seems reasonable. No action required.